### PR TITLE
Update to magick 7 in alpine.

### PR DESCRIPTION
**Pull request by googollee**
As we switched to `alpine`, we could update `magick` commands in the new way.

### CODE REVIEW

File: api/scanner/media_encoding/executable_worker/executable_worker.go [hunk 1]
```diff
@@ -37,17 +37,17 @@ func newMagickWorker() *MagickWorker {
 		return nil
 	}
 
-	path, err := exec.LookPath("convert")
+	path, err := exec.LookPath("magick")
 	if err != nil {
-		log.Println("Executable worker not found: ImageMagick convert")
+		log.Println("Executable worker not found: magick")
```
Comment (kkovaletp) [inline api/scanner/media_encoding/executable_worker/executable_worker.go:42]: While the `magick` executable name better correlates with the 'ImageMagick', I'm not sure that it is intuitive enough for users to be mentioned in log messages as is...
This comment is more like a discussion starter than a change request...

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Tests
 
 on:
   push:
-    branches: [master]
+    branches: ['*']
```
Comment (kkovaletp) [inline .github/workflows/tests.yml:5]: A standard reminder for testing changes)

**Review by kkovaletp — COMMENTED** [review]
Just 1 topic to discuss.
Apart from that, I don't have any other questions. This is a small PR)
Why wasn't the `magick` executable used in the Debian image?

**Comment by kkovaletp** [comment]
Ah, 1 more question:
The PR is named as 'Update to magic 7', but I don't see the version change in the changed files... How does it work then?)

**Comment by googollee** [comment]
Debian testing only provides ImageMagick 6. I believe we discussed this before. ImageMagick 6 -> 7 changes the way to call the command. With 7, all commands of ImageMagick are under `magick` command, e.g. `convert` -> `magick convert`.

Alpine provides ImageMagick 7, and I want to follow the new way here. I didn't do it in the previous PR because 7 also provides commands for the legacy way and I didn't want to change too many things in one PR.

File: api/scanner/media_encoding/executable_worker/executable_worker.go (hunk 1 above)
Comment (googollee) [reply api/scanner/media_encoding/executable_worker/executable_worker.go:42]: Here `magick` is the main command of ImageMagick. It follows the way of `ffmpeg` below.

With ImageMagick 6, a command like `convert` isn't able to refer ImageMagick easily, so I use `ImageMagick convert`. Now `magick` is really identifiable and no need `ImageMagick` as a prefix.

**Comment by kkovaletp** [comment]
Ok, I see: the ImageMagick's executable was updated together with the base image. Here you update only the way, it is called - from a legacy one to a modern one.

File: .github/workflows/tests.yml (hunk 2 above)
Comment (kkovaletp) [reply .github/workflows/tests.yml:5]: Revert the testing code and I'll approve the PR

Comment (googollee) [reply .github/workflows/tests.yml]: Finished testing and reverted workflows.

**Review by kkovaletp — APPROVED** [review]
LGTM

**Pull request merged**